rust/osm2lanes-web/src/main.rs

```diff
@@ -8,10 +8,9 @@ use yew::prelude::*;
 
 mod draw;
 
-use osm2lanes::{
-    lanes_to_tags, tags_to_lanes_with_warnings, DrivingSide, Lane, LanePrintable, Lanes,
-    LanesToTagsConfig, Locale, Tags,
-};
+use osm2lanes::{lanes_to_tags, tags_to_lanes, LanesToTagsConfig, TagsToLanesConfig};
```

Comment:
Curious what motivates this split? If the namespace feels like it's getting too big, maybe we shouldn't publish a flattened one, but have modules maybe?

Reply:
I'll be honest, I don't know how to do this properly in rust and have not yet looked it up.

Maybe I'll make it the topic of my next refactoring :)